packages/components/src/menu/docs/Dividers.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button, Divider, Menu, MenuItem, MenuTrigger } from "@hopper-ui/components";
+import { KebabIcon } from "@hopper-ui/icons";
+
+export default function Example() {
+    return (
+        <MenuTrigger>
+            <Button variant="secondary" aria-label="Actions for selected resource">
+                <KebabIcon />
+            </Button>
+            <Menu>
+                <MenuItem>Favorite</MenuItem>
+                <MenuItem>Edit</MenuItem>
+                <Divider />
```

Comment:
i think the default style of divider should have margin top/bottom, a little one. I don't know if there is figma for this, but in orbiter there is one and it looks better when hovering items